web/views.py

```
@@ -7103,3 +7128,112 @@ def users_list(request: HttpRequest) -> HttpResponse:
+                # Different vote type, so update the vote
+                vote.vote_type = vote_type
+                vote.save()
+
+        # After processing the vote, redirect back to the topic page
+        return redirect("forum_topic", category_slug=reply.topic.category.slug, topic_id=reply.topic.id)
+
+    except ForumReply.DoesNotExist:
+        messages.error(request, "Reply not found")
+        return redirect("forum_categories")
+
+    except ForumReply.DoesNotExist:
```

Ruff (0.8.2)

try-except block with duplicate exception `ForumReply.DoesNotExist`

(B025)